docs/schema/animated-properties/animated.json

```diff
@@ -0,0 +1,13 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "Animated",
+    "description": "Whether the property is animated",
+    "type": "object",
+    "properties": {
+        "a": {
+            "$ref": "#/$defs/helpers/int-boolean",
+            "default": 0
+        }
+    },
+    "additionalProperties": false
```

Comment:
wouldn't `additionalProperties` cause issues?

Reply:
Agree, to be safe and avoid unexpected problems let me remove them